ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -314,7 +306,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     ./nnstreamer_sink_example &>> temp.log
     result+=$(save_consumer_msg $?)
 
-    # App (Consumer): ./nnstreamer_sink_example.py to convert video images to tensor,
+    # App (Consumer): ./nnstreamer_sink_example_play to convert video images to tensor,
```

Comment:
This comment is to explain how to run a python code of nnstreamer_sink_example. Why do you want to change the file name?

Reply:
It's okay. Synced.